### PR TITLE
[yeelight] Added support Ceiling 1 with Night Mode

### PR DESCRIPTION
**Pull request by DragonNP**
Signed-off-by: Nikita Pogudalov <dragonnp@ya.ru>

Support Yeelight Ceiling Light 1 with night mode

- [yeelight] Added Ceiling 1 support
- Add support for Yeelight Ceiling Light with night mode
- In this update, you can add Yeelight Ceiling 1 devices with night mode. Updated README

You can download the JAR file from this link: https://drive.google.com/open?id=1dtmXRw_gESHKkrOTgbh-lQzMZnflhU0w

### CODE REVIEW

File: bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+public class YeelightCeilingWithNightHandler extends YeelightHandlerBase {
+
+    public YeelightCeilingWithNightHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        handleCommandHelper(channelUID, command, "Handle ceiling ambient light command");
+    }
```
Comment (cpmeister) [inline bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java:38]: I don't think you need to override this. YeelightCeilingHandler already implements this for you, unless you want to fix the string instead.

File: bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.openhab.binding.yeelight.internal.lib.device.DeviceStatus;
+import org.openhab.binding.yeelight.internal.lib.enums.ActiveMode;
+
+/**
+ * The {@link YeelightCeilingWithNightHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author DragonNP - Initial contribution
+ */
+public class YeelightCeilingWithNightHandler extends YeelightHandlerBase {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java:29]:
```suggestion
public class YeelightCeilingWithNightHandler extends YeelightCeilingHandler {
```

File: bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/YeelightHandlerFactory.java [hunk 3]
```diff
@@ -29,6 +29,7 @@
  * The {@link YeelightHandlerFactory} is responsible for returning supported things and handlers for the devices.
  *
  * @author Coaster Li - Initial contribution
+ * @author DragonNP - Added YeelightCeilingWithNightHandler for Ceiling 3
```
Comment (cpmeister) [inline bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/YeelightHandlerFactory.java:32]: Also, please use your real name in the authoring tags

**Comment by DragonNP** [comment]
Thanks you!

**Pull request merged**